sql/parser/builtins.go

```diff
@@ -761,3 +774,12 @@ func datumToRawString(datum Datum) (string, error) {
 
 	return "", fmt.Errorf("argument type unsupported: %s", datum.Type())
 }
+
+func round(x float64, n int64) (Datum, error) {
```

Comment:
In this case it's used by two builtins with different type lists: the `round(x float, ndigits int)` and `round(x float)`.